src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -53,11 +53,43 @@
         }
 
     ],
-
+    "Categories": [
+        {
+            "Id": "General",
+            "DisplayName": "General",
+            "Order": 10
```

Comment:
Just preference that it would be easier to inject a new category in the future without changing all of the other numbers. I also don't want recipe authors to think there can't be gaps in the numbers.